Sources/web3swift/Utils/EIP/EIP712.swift

```diff
@@ -127,7 +127,7 @@ fileprivate extension EIP712Hashable {
             }
             return typeName + " " + key
         }
-        return name + "(" + parametrs.joined(separator: ",") + ")"
+        return name + "(" + parameters.joined(separator: ",") + ")"
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.